targets/TARGET_WICED/wiced_port/WICED/network/mbed/WWD/wwd_buffer.c

```diff
@@ -0,0 +1,98 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#if defined (IAR_TOOLCHAIN)
```

Comment:
should use IAR identifier rather here

why a header file inclusion is based on the toolchain?

Reply:
This file had a lot of revisions, I'll see if I can remove it